src/test/resources/bundle_r1.2/GAVIN_calibrations_r1.2.tsv

```diff
@@ -0,0 +1,3500 @@
+Gene	Category	Chr	Start	End	NrOfPopulationVariants	NrOfPathogenicVariants	NrOfOverlappingVariants	NrOfFilteredPopVariants	PathoMAFThreshold	PopImpactHighPerc	PopImpactModeratePerc	PopImpactLowPerc	PopImpactModifierPerc	PathoImpactHighPerc	PathoImpactModeratePerc	PathoImpactLowPerc	PathoImpactModifierPerc	PopImpactHighEq	PopImpactModerateEq	PopImpactLowEq	PopImpactModifierEq	NrOfCADDScoredPopulationVars	NrOfCADDScoredPathogenicVars	MeanPopulationCADDScore	MeanPathogenicCADDScore	MeanDifference	UTestPvalue	Sens95thPerCADDThreshold	Spec95thPerCADDThreshold
+DEFAULT									0.003456145																				15
```

Comment:
I would add this line as a header line at the beginning of the file.

Reply:
Discuss with Joeri